public/apps/configuration/app-router.tsx

```diff
@@ -40,10 +40,12 @@ import { UserList } from './panels/user-list';
 import { Action, ResourceType, RouteItem, SubAction } from './types';
 import { buildHashUrl, buildUrl } from './utils/url-builder';
 
+const LANDING_PAGE_URL = '/getstarted';
```

Comment:
Can we move `getstarted` to `ResourceType` enum and use `buildUrl()` to be consistent with other?

Reply:
I see both have valid points. One is for style consistent and one is for semantic clarity.

If I am the author, I would prefer separating getstarted from others since it is indeed an instruction page and is not related to any json object from backend.